src/oas3/accessors.ts

```diff
@@ -1,17 +1,16 @@
 import { get } from 'lodash';
 import { OpenAPIObject, OperationObject, SecuritySchemeObject } from 'openapi3-ts';
 
-import { mapToKeys } from '../utils';
+import { isObject, mapToKeys } from '../utils';
```

Comment:
Would any of the following work?

- https://lodash.com/docs/4.17.14#isObject
- https://lodash.com/docs/4.17.14#isObjectLike
- https://lodash.com/docs/4.17.14#isPlainObject

Reply:
Oh, I thought `isObject` wasn't a type-guard, but it is!
Going to use it.